pkgs/top-level/all-packages.nix

```diff
@@ -36247,6 +36247,10 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Cocoa;
   };
 
+  unison-fsmonitor = callPackage ../by-name/un/unison-fsmonitor/package.nix {
+    inherit (darwin.apple_sdk.frameworks) CoreServices;
+  };
+
```

Comment:
This is confusing. `pkgs/by-name` is intended to automatically include packages and should eliminate the need to manually add them to `all-packages.nix`. Maybe this could also break something in the future.

Please just use `darwin.apple_sdk.frameworks.CoreServices` in your package.